cmd/defaults.yaml

```diff
@@ -39,16 +39,15 @@ Telemetry:
 
 # Port ZITADEL will listen on
 Port: 8080 # ZITADEL_PORT
-# Port ZITADEL is exposed on, it can differ from port e.g. if you proxy the traffic
-# !!! Changing this after the initial setup breaks your system !!!
+# ExternalPort is the port on which end users access ZITADEL.
+# It can differ from Port e.g. if a reverse proxy forwards the traffic to ZITADEL
+# Search the ZITADEL docs page "External ZITADEL Access" more information.
```

Comment:
why don't we link here to the docs directly?

Reply:
Will do 👍

Reply:
done